projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.ts

```diff
@@ -69,4 +69,10 @@ export class DatatableCellBadgesComponent implements OnInit, OnDestroy {
       this.appExtensionService.runActionById(badge.actions?.click, this.node);
     }
   }
+
+  onKeyPress(event: KeyboardEvent, badge: Badge) {
```

Comment:
You can also do `(keypress.enter) = onKeyPress(badge)` in the html. That way you won't have to pass the event object to the function, and won't have to do the if check